docs/src/app/docs/[[...slug]]/page.tsx

```diff
@@ -36,7 +38,15 @@ export default async function Page(props: { params: Promise<{ slug?: string[] }>
                     }}
                 />
             </DocsBody>
-            {hasCategory && <DocsCategory page={page} from={docsSource} />}
+            {hasCategory && (
+                <Cards>
+                    {getPageTreePeers(docsSource.pageTree, page.url).map(peer => (
+                        <Card key={peer.url} title={peer.name} href={peer.url}>
+                            {peer.description}
+                        </Card>
+                    ))}
+                </Cards>
+            )}
```

Comment:
`fumadocs-ui` recently removed `DocsCategory` and recommends doing this instead.